nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -465,7 +465,11 @@ public void compareGenericTypeParameterNullabilityForCall(
     if (!config.isJSpecifyMode()) {
       return;
     }
-    for (int i = 0; i < formalParams.size(); i++) {
+    int n = formalParams.size();
+    if (isVarArgs) {
+      n = n - 1; // don't check for the last argument if
```

Comment:
If what? I mean, it's fairly clear, but lets complete the comment. Maybe it should be a comment above like:

```
// If the last argument is var args, don't check it now, it will be checked against
// all remaining actual arguments in the next loop.
```